### DISCUSSION TITLE
how do I make sure that only the window of the working form is launched?

### DISCUSSION THREAD

**Discussion by SaBond0** [AvaloniaUI/Avalonia#14208] [Q&A] [answered]
at startup, the console starts, or some kind of frame that has its own size and takes the entire size of the form, even if I reduce the form, it will still take more, even though I don't see these fields in the editor anymore, here are screenshots of what I mean.and also takes the colors as in the visual studio theme, I don't need such settings, I only want to see the shape
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/84dbcefe-35be-4400-971a-f9ee2daeb00e)
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/66a0c1af-5630-4573-ad97-bfc3eebae6dd)

**Comment by timunie** [comment]
@SaBond0 are you asking the same question elsewhere? Please use one thread only. 

Things to check: 
1. Make sure your `csproj` file has `WinExe` as output type
2. Make sure to set Window Width and not DesignWidht. You can also use SizeToContent. 
3. In App.axaml set `RequestedThemeVariant="Light"` to get the light theme enabled always. By default Avalonia follows your system settings.

**Reply by SaBond0** [reply]
I checked everything except for 2 points, when I change it gives an error here is an example
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/e23a50e5-9381-4e1c-9942-c738dd621f6b)

**Reply by StefanKoell** [reply]
I may be wrong but I think you need d:Width AND d:Height in your XAML and optionally d:DesignWidth and d:DesignHeight in there.

**Reply by timunie** [reply, marked as answer]
remove the leading "d:" it is the design-time namespace.

**Reply by SaBond0** [reply]
I probably chose the right answer early, because it didn't turn out exactly what I wanted, how to make the pop-up window have no extra frames and empty spaces besides the form itself?
![Uploading image.png…]()

**Reply by timunie** [reply]
Check WindowDecorations properties and friends https://docs.avaloniaui.net/docs/tutorials/music-store-app/creating-a-modern-looking-window

**Comment by thevortexcloud** [comment]
Alright, I think I understand. You are calling the previewer "the console".  The console typically refers to `System.Console`. This explains a lot of my earlier confusion. The previewer is just for seeing how your app looks based on your current XAML and at various different sizes. It has no real relation to your runtime window size.

I suggest reading the basics tutorial on the docs site. 

https://docs.avaloniaui.net/ru/docs/basics/user-interface/introduction-to-xaml

**Reply by SaBond0** [reply]
I'll take a good look, thanks
